lab7/ChristmasTree.java

```diff
@@ -0,0 +1,116 @@
+package javalab.lab7;
+
+public class ChristmasTree {
+    public int height = 0;
+    public String producer = "";
+    public float price = 0;
+    public String material = "";
+
+    private String color = "";
```

Comment:
are you sure that it is important to set empty value? 
read about default values and better, if you need to set some real valuer (not 0 for int or "" for String) put that logic to default constructor or initialization bloc

Reply:
removed